docs/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG PYTHON_VERSION=3.8.1-alpine3.11
+
+FROM python:${PYTHON_VERSION} as builder
+
+ENV PYTHONUNBUFFERED 1
+
+# Set build directory
+WORKDIR /wheels
+
+# Copy files necessary
+COPY ./requirements.txt .
+
+# Perform build and cleanup artifacts
+RUN \
+  apk add --no-cache \
+  git \
+  git-fast-import \
+  && apk add --no-cache --virtual .build gcc musl-dev \
+  && python -m pip install --upgrade pip \
+  && pip install -r requirements.txt \
+  && apk del .build gcc musl-dev \
+  && rm -rf /usr/local/lib/python3.8/site-packages/mkdocs/themes/*/* \
+  && rm -rf /tmp/*
+
+
+
+# Set final MkDocs working directory
+WORKDIR /docs
+
+# Expose MkDocs development server port
+EXPOSE 8000
+
+# Start development server by default
+ENTRYPOINT ["mkdocs"]
+CMD ["serve", "--dev-addr=0.0.0.0:8080"]
```

Comment:
listening on `0.0.0.0` spits out this warning
```
WARNING  -  Config value: 'dev_addr'. Warning: The use of the IP address '0.0.0.0' suggests a production environment or the use of a proxy to connect to the MkDocs server. However, the MkDocs' server is intended for local development purposes only. Please use a third party production-ready server instead.
```

Is it a good idea to listen on all interfaces in development?